data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2098,20 +2169,34 @@
             message= _ "Who are you that dares to venture down these tunnels?"
         [/message]
 
-        [message]
-            speaker=unit
-            message= _ "I... I am one of Tallin’s men..."
-        [/message]
+        [if]
+	        [variable]
+		        name=unit.id
+		        equals="Tallin"
+            [/variable]
+            [then]
+                [message]
+                    speaker=Tallin
+                    message= _ "I am Tallin."
+                [/message]
+                [/then]
```

Comment:
The last commit introduced an extra level of indentation here, it should be removed.